cloudinit/net/dhcp.py

```diff
@@ -92,9 +92,9 @@ def obtain_lease(self):
         nmap = {'interface': 'interface', 'ip': 'fixed-address',
                 'prefix_or_mask': 'subnet-mask',
                 'broadcast': 'broadcast-address',
-                'static_routes': 'rfc3442-classless-static-routes',
+                'static_routes': ['rfc3442-classless-static-routes', 'classless-static-routes'],
```

Comment:
Do you have a reference lease file or docs on the static routes fields and the format?  When I looked at this last, the format found in centos was an internal dhclient layout, and I don't see that being parsed in this PR, so it appears that there's something else in the centos lease file that represents the rfc3442 values?

Reply:
Hi @raharper ,

I just grabbed this (but changed some values in there).

`lease {
  interface "eth0";
  fixed-address 192.168.128.12;
  option subnet-mask 255.255.255.0;
  option routers 192.168.128.1;
  option dhcp-lease-time 86400;
  option dhcp-message-type 5;
  option domain-name-servers 191.168.128.1;
  option dhcp-server-identifier 192.168.128.3;
  option interface-mtu 1500;
  option dhcp-renewal-time 39722;
  option classless-static-routes 24.191.128.168 192.168.128.1,32.169.254.169.254 192.168.128.1,0 192.168.128.1;
  option broadcast-address 192.168.128.255;
  option dhcp-rebinding-time 72122;
  option host-name "host-192-168-128-12";
  option domain-name "openstacklocal";
  renew 2 2019/12/03 05:31:19;
  rebind 2 2019/12/03 16:03:04;
  expire 2 2019/12/03 20:01:02;
}
`

Notice the name and the separated value.    Also, sorry for the pipeline run, I had failures locally that didn't seems to correlate with changes in my codes, so I wondered if the CI would fail.

Reply:
Now, I'm wondering why I have failures locally...

Reply:
BTW, the reference comes from a Centos 7 up-to-date.  I didn't look at centos 6 or redhat OS.

Reply:
Oh, I just got what you meant, the format is `.` and not `,`.  Let me try to see if we get that.

Reply:
Hi @raharper, please let me know if this is resolved.  I added the test case just for red hat format over the parser logic for more than one route, another one for the lease to be read and one for the lease to be interpreted ( I used 2 of the previously present tests and adapted them).